cmd/telemeter-server/main.go

```diff
@@ -42,6 +37,10 @@ import (
 	"github.com/openshift/telemeter/pkg/metricfamily"
 	"github.com/openshift/telemeter/pkg/receive"
 	"github.com/openshift/telemeter/pkg/server"
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
I think this was done by mistake. These imports should be grouped with the non telemeter imports above